changelog_entry.yaml

```
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - Pennsylvania income deduction
```

Though I think it's more than this - revise to capture the full scope of the PR
```suggestion
    - Pennsylvania TANF income deduction.
```